src/plugins/mer/mersettings.h

```diff
@@ -56,7 +58,21 @@ class MerSettings : public QObject
     static bool isAskBeforeClosingVmEnabled();
     static void setAskBeforeClosingVmEnabled(bool enabled);
 
+    static Utils::FileName deviceModelsFileName();
+    static Utils::FileName globalDeviceModelsFileName();
+
+    enum EmulatorDeviceModelType {
+        EmulatorDeviceModelSdkProvided,
+        EmulatorDeviceModelUserProvided,
+        EmulatorDeviceModelAll
+    };
+    static MerEmulatorDeviceModel::Map deviceModels(EmulatorDeviceModelType type
+                                                    = EmulatorDeviceModelType::EmulatorDeviceModelAll);
+    static bool isDeviceModelStored(const MerEmulatorDeviceModel &model);
```

Comment:
Ideally we should also move isDeviceModelStored to the next commit. This would also allow to move MerEmulatorDeviceModel::operator== there.

Reply:
Done.